### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularHome_AI4W: No main landmark is defined on the "Home" page

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42341]
![42341_A11y_ NET Core_ASP NETCore_AngularHome_AI4W](https://user-images.githubusercontent.com/86710491/175480595-a6f2c094-83ce-47f3-9597-f1112b32d777.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopWeb;#Win11;#Benchmark;#Element:Landmark;#AI4W;#AIManual;#ceaccreviewed;#A11ycaioh;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Test the home page.
10. Run AI4W for amd check for landmarks.

### Actual:
No main landmark is defined on the "Home" page

**This issue is observed throughout the application for all the Home pages.**

### Expected:
Main landmark should be defined on the "Home" page

### User Impact:
If main landmark is not define on the page than users dependent on the screen reader will get difficulties to interact the page because when users will navigate in landmark mode, they will get difficulties to understand the purpose of the page.

**Comment by SteveSandersonMS** [comment]
Fixed in https://github.com/dotnet/spa-templates/pull/70

**Comment by mkArtakMSFT** [comment]
#Closed;
